api/tests/integration/UserTest.php

```diff
@@ -222,4 +217,21 @@ public function testDeleteOneByGuestUser()
         $this->assertResponseStatus(403);
         $this->assertEquals($rowCount, User::count());
     }
+
+    public function testResetPasswordMail()
+    {
+        $this->clearMessages();
+        $user = $this->createUser('guest');
+        $token = $this->tokenFromUser($user);
+
+        $this->delete('/users/'.$user->user_id.'/password', [], [
+            'HTTP_AUTHORIZATION' => 'Bearer '.$token
+        ]);
+
+        $mail = $this->getLastMessage();
+
+        $this->assertResponseStatus(202);
+        $this->assertResponseHasNoContent();
+        $this->assertContains('Password', $mail->subject);
+    }
```

Comment:
@xiphiaz Adding that. FIY, `AuthTest::testTokenInvalid` does already test that functionality. Doesn't hurt to test it twice I guess.